chain/chain/src/types.rs

```diff
@@ -748,11 +748,12 @@ pub trait RuntimeAdapter: Send + Sync {
     /// Build receipts hashes.
     // Due to borsh serialization constraints, we have to use `&Vec<Receipt>` instead of `&[Receipt]`
     // here.
-    fn build_receipts_hashes(
-        &self,
-        receipts: &[Receipt],
-        shard_layout: &ShardLayout,
-    ) -> Vec<CryptoHash> {
+    fn build_receipts_hashes(&self, receipts: &[Receipt], epoch_id: &EpochId) -> Vec<CryptoHash> {
+        let shard_layout = //{
+            //let epoch_id = self.get_epoch_id_from_prev_block(&prev_block_hash).unwrap();
+            self.get_shard_layout(&epoch_id).unwrap();
+        //};
+        //self.build_receipts_hashes(receipts, &shard_layout)
```

Comment:
Some left-over comments? I think it's best to avoid commented-out code in the repository, even if it's only temporarily. Such comments tend to stay for a long time more often than not.